org.telegram.desktop.json

```diff
@@ -187,12 +81,8 @@
                 {
                     "type": "git",
                     "url": "https://github.com/telegramdesktop/tdesktop.git",
-                    "tag": "v2.1.1",
-                    "commit": "07c8aae225e2f04600a7d5411047d30f4ca5f51c"
-                },
-                {
-                    "type": "patch",
-                    "path": "patches/Fix-decoration-applying.patch"
```

Comment:
Remove the patch file, too, if it's not needed.

Reply:
Oh, looks like my vscode doesn't delete files again

Thanks